Makefile

```diff
@@ -132,55 +132,67 @@ ci/cli/push: ci/cli/build
 # uidp {
 ## packages the UIDP helm chart using the UIDP overrides file, which by default is uidp-overwrites.yaml
 helm/uidp/package:
-	helm package ./build/inspr-stack/subcharts/uidp -o charts -f ${UIDP_VALUES} -n ${K8S_NAMESPACE}
+	helm package ./build/uidp
 
 ## lints the UIDP helm chart using the UIDP overrides file.
 helm/uidp/lint:
-	helm lint ./build/inspr-stack/subcharts/uidp -o charts -f ${UIDP_VALUES} -n ${K8S_NAMESPACE}
+	helm lint ./build/uidp
 
 ## runs the UIDP helm chart tests using the UIDP overrides file,
 helm/uidp/test:
-	helm test ./build/inspr-stack/subcharts/uidp -o charts -f ${UIDP_VALUES} -n ${K8S_NAMESPACE}
+	helm test ${RELEASE_NAME}-uidp -n ${K8S_NAMESPACE}
 
 ## installs the uidp helm chart to the K8S_NAMESPACE using the uidp overrides file.
 helm/uidp/install:
-	helm install ./build/inspr-stack/subcharts/uidp -o charts -f ${UIDP_VALUES} -n ${K8S_NAMESPACE}
+	helm upgrade -i ${RELEASE_NAME}-uidp ./build/uidp -f ${UIDP_VALUES} -n ${K8S_NAMESPACE}
+
+## delete a helm release
+helm/uidp/uninstall:
+	helm uninstall ${RELEASE_NAME}-uidp -n ${K8S_NAMESPACE}
 # }
 
 # insprd {
 ## packages the INSPRD helm chart using the INSPRD overrides file, which by default is insprd-overwrites.yaml
 helm/insprd/package:
-	helm package ./build/inspr-stack/subcharts/insprd -o charts -f ${INSPRD_VALUES} -n ${K8S_NAMESPACE}
+	helm package ./build/insprd
 
 ## lints the INSPRD helm chart using the INSPRD overrides file.
 helm/insprd/lint:
-	helm lint ./build/inspr-stack/subcharts/insprd -o charts -f ${INSPRD_VALUES} -n ${K8S_NAMESPACE}
+	helm lint ./build/insprd
 
 ## runs the INSPRD helm chart tests using the INSPRD overrides file,
 helm/insprd/test:
-	helm test ./build/inspr-stack/subcharts/insprd -o charts -f ${INSPRD_VALUES} -n ${K8S_NAMESPACE}
+	helm test ${RELEASE_NAME}-insprd -n ${K8S_NAMESPACE}
 
 ## installs the insprd helm chart to the K8S_NAMESPACE using the insprd overrides file.
 helm/insprd/install:
-	helm install ./build/inspr-stack/subcharts/insprd -o charts -f ${INSPRD_VALUES} -n ${K8S_NAMESPACE}
+	helm upgrade -i ${RELEASE_NAME}-insprd ./build/insprd -f ${INSPRD_VALUES} -n ${K8S_NAMESPACE}
```

Comment:
I don't particularly like having the `insprd/install` doing the `helm upgrade`. Would be best to make a `helm/insprd/upgrade` command

Besides does the `helm upgrade` work without the `--install` work?